sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -113,7 +113,10 @@ def __init__(self, message=None, response=None, **kwargs):
         self.response = response
         if response:
             self.reason = response.reason
-            self.status_code = response.status_code
+            try:
```

Comment:
Shouldn't the response be a unified/pipeline response?

Reply:
Yes - this will be fixed as soon as I merge the latest changes from master